src/routes/events/containers/EventsContainer.js

```diff
@@ -126,7 +126,7 @@ class EventsContainer extends React.Component<ContainerPropsType> {
 const refresh = (refreshProps: RefreshPropsType, dispatch: Dispatch<StoreActionType>) => {
   const { navigation, cityCode, language, path } = refreshProps
   const navigateToEvent = createNavigateToEvent(dispatch, navigation)
-  navigateToEvent({ cityCode, language, path, forceUpdate: true, key: navigation.getParam('key') })
+  navigateToEvent({ cityCode, language, path, forceUpdate: true, key: navigation.state.key })
```

Comment:
Not sure about this change, but if it works it is probably oke :)

Reply:
Yeah, having the navigation in the navigation params doesn't make a lot of sense, since it's available through the navigation state